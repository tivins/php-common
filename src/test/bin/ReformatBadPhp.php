<?php

use Tivins\Core\System\Terminal;

require 'vendor/autoload.php';

$a = array(
    'test'
    => 2
);

var_dump($a);
Terminal::sleep(1);
$a['test']++;
var_dump($a);
Terminal::sleep(.2);
