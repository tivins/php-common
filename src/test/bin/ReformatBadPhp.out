array(1) {
  ["test"]=>
  int(2)
}
Sleeping 1.0 seconds...
[1A[KSleeping 1.0 seconds...
[1A[KSleeping 1.0 seconds...
[1A[KSleeping 1.0 seconds...
[1A[KSleeping 1.0 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.9 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.8 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.7 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.6 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.5 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.4 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.3 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.0 seconds...
[1A[KSleeping 0.0 seconds...
[1A[KSleeping 0.0 seconds...
[1A[KSleeping 0.0 seconds...
[1A[KSleeping 0.0 seconds...
[1A[Karray(1) {
  ["test"]=>
  int(3)
}
Sleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.2 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.1 seconds...
[1A[KSleeping 0.0 seconds...
[1A[KSleeping 0.0 seconds...
[1A[KSleeping 0.0 seconds...
[1A[KSleeping 0.0 seconds...
[1A[KSleeping 0.0 seconds...
[1A[K